apps/dashboard/src/components/primitives/field-editor/field-editor.tsx

```diff
@@ -0,0 +1,232 @@
+import { EditorView } from '@uiw/react-codemirror';
```

Comment:
How about calling this `ControlField` or `ControlInput`? I prefer the latter as it is an input for Novu Controls. It maps better with the domain.